assets/javascripts/discourse/initializers/discourse-reactions.gjs

```diff
@@ -128,21 +128,26 @@ function initializeDiscourseReactions(api) {
 
         get label() {
           const count = this.notification.data.count;
-          const username = this.username;
+          const nameOrUsername = this.siteSettings.prioritize_full_name_in_ux
+            ? this.notification.data.display_name ||
+              this.notification.data.username
```

Comment:
```suggestion
              this.username
```

Reply:
Any reason this shouldn't be `this.notification.data.display_username`? Turns out `.username` doesn't exist on the notification object, so the way I had it wouldn't have worked anyway.

Reply:
Yes `this.notification.data.username` doesn't exist, however `this.username` is just a [getter](https://github.com/discourse/discourse/blob/f744852f105f2ac355af843b646fb2362cacc46d/app/assets/javascripts/discourse/app/lib/notification-types/base.js#L113-L115) that returns `this.notification.data.display_username` with `formatUsernaem()` applied on it. Is `this.username` not returning anything in your tests?

Reply:
Ahh gotcha thank you! Nah it worked :)